src/librustc_mir/interpret/mod.rs

```diff
@@ -1,5 +1,10 @@
 //! An interpreter for MIR used in CTFE and by miri
 
+#[macro_export]
+macro_rules! err {
+    ($($tt:tt)*) => { Err($crate::interpret::EvalErrorKind::$($tt)*.into()) };
+}
```

Comment:
Oh, I see. I did not know that. I got used to ordering not mattering in Rust. ;)